.github/workflows/back-dev.yml

```
@@ -9,61 +9,68 @@ env:
-        echo "Fetched TASK_DEF: $TASK_DEF"
-        aws ecs update-service \
-          --cluster $ECS_CLUSTER \
-          --service $ECS_SERVICE \
-          --task-definition $NEW_TASK_ARN \
-          --force-new-deployment
\ No newline at end of file
+      - name: Send Slack start_message
+        run: |
+          curl -X POST -H 'Content-type: application/json' \
+          --data '{"text":"✅ 배포가 완료되었습니다!"}' \
+          ${{ secrets.SLACK_WEBHOOK_URL }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)